src/sgame/sg_buildable.cpp

```diff
@@ -1702,6 +1702,23 @@ itemBuildError_t G_CanBuild( gentity_t *ent, buildable_t buildable, int /*distan
 		level.numBuildablesForRemoval = 0;
 	}
 
+	int max_miners = g_maxMiners.Get();
+	if ( max_miners >= 0 && ( buildable == BA_H_DRILL || buildable == BA_A_LEECH ) )
+	{
+		int miners = 0;
+		ForEntities<MiningComponent> ( [&](Entity& entity, MiningComponent& )
+		{
+			if ( Entities::IsAlive(entity) && G_OnSameTeam( entity.oldEnt, ent ) )
+			{
+				miners++;
+			}
+		});
+		if ( miners >= max_miners )
+		{
+			return ent->client->pers.team == TEAM_HUMANS ? IBE_NOHUMANBP : IBE_NOALIENBP;
```

Comment:
just a note for later, probably worth a new issue (that means: I don't mind if this is not fixed, but I'd prefer a `TODO 0.54 bla bla` comment): IBE_NO...BP does not suits well the real error.

Reply:
We use the `TODO(0.54)` notation for things that have to go in the next release due to compatibility breakage. We decided to allow changing the cgame<>sgame protocol at any time, so that does not apply.